src/gui/sdl_mapper.cpp

```diff
@@ -2760,8 +2760,6 @@ void BIND_MappingEvents()
 			 */
 			if ((event.window.event == SDL_WINDOWEVENT_RESIZED) ||
 			    (event.window.event == SDL_WINDOWEVENT_RESTORED)) {
-				GFX_UpdateDisplayDimensions(event.window.data1,
```

Comment:
Are we sure this is Android-only? What about convertible laptops?

Reply:
Reworked it a bit so it only affects the special Android workaround for the mapper. I'm reluctant to keep workaround's that we cannot test and are unsure about. I think it's best to get rid of them and only reintroduce them if people report problems if we have no way to test them.